### PR TITLE
Fix #4, use `sysctl memsize` for Mac benchmark

### LINKED ISSUE #4: Mac_BSD_resources underestimates RAM size
[`Mac_BSD_resources.py`](https://github.com/SeattleTestbed/resource/blob/master/Mac_BSD_resources.py#L55) uses `sysctl hw.physmem` to find the size of RAM. Unfortunately, the info is off by a factor of 4 on a MacBook Pro 8,2 running Mac OS X 10.6.8 on which I'm testing this. (At least we are *underestimating* the size, so this only hurts the Seattle install, but not the resource donor.)

`sysctl hw.memsize` gives the correct memory size, 8 GB here.

See also https://discussions.apple.com/thread/1753088?start=0&tstart=0

We should test what the different `hw.` keys return on a couple of machines, and then decide on a fix.

### PR DESCRIPTION
**Pull request by aaaaalbert**

### CODE REVIEW

File: Mac_BSD_resources.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def measure_resources():
 
   # Next, get physical memory (RAM)
```
Comment (vladimir-v-diaz) [inline Mac_BSD_resources.py]: Total physical memory?  In bytes?

File: Mac_BSD_resources.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def measure_resources():
...
   phys_mem = "unable to find size of physical memory"
```
Comment (vladimir-v-diaz) [inline Mac_BSD_resources.py]: Probably not worth changing it now, but it's odd that some of these resource values can be either a string or an integer.

File: Mac_BSD_resources.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ def measure_resources():
...
-  pipe = getShellPipe("sysctl hw.physmem")
+  pipe = getShellPipe("sysctl hw.memsize")
   for line in pipe:
     line_s = line.split(" ")
```
Comment (vladimir-v-diaz) [inline Mac_BSD_resources.py]: You may even provide an example (hw.memsize: 8589934592), or use the expected `hw.memsize` at the start to parse the needed value.

**Comment by vladimir-v-diaz** [comment]
You'll likely want to fix the following in a separate pull request, but this module contains many unused lines that are commented out.

File: Mac_BSD_resources.py (hunk 3 above)
Comment (aaaaalbert) [inline Mac_BSD_resources.py]: Updated the commit accordingly. Thanks!

File: Mac_BSD_resources.py (hunk 2 above)
Comment (aaaaalbert) [inline Mac_BSD_resources.py]: Yeah, that approach leaves a lot to be desired.

**Pull request merged**